### DISCUSSION TITLE
How can I disable RGB convertion

### DISCUSSION THREAD

**Discussion by musen34** [tailwindlabs/tailwindcss#7475] [Help]
How can I disable RGB convertion? I want to use directly hex. Its converting like RGB by default.. I dont know how can I disable RGB.

my configurations;

module.exports = {
  mode: 'jit',
  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
  darkMode: 'class', // false or 'media'
  theme: {
    extend: {
      colors: {
        brand: {
          bg: '#1C2F44',
          blue50: '#71CEDC',
          blue100: '#20344A',
          blue150: '#00adee',
          blue250: '#334960',
          blue300: '#475993',
          gray100: '#36485D',
          gray150: '#BFBFBF',
          darkmode: '#161B22',
          gray200: '#2E445C',
          gray250: '#9CA7B5',
          gray400: '#263A4F',
          gray550: '#233950',
          gray700: '#9AAFC6',
          gray800: '#CED0D7',
          gray900: '#35485B',
          green50: '#91DC5A',
          green100: '#71CEDC',
          red50: '#FF5A51',
          orange50: '#FFB000'
        }
      },

**Comment by rflw** [comment]
It's probably not possible without external library to convert rgba to hex.

**Comment by musen34** [comment]
its should be possible, because its converting by default @adamwathan ?

**Reply by musen34** [reply]
When I disable bgopacity parameter to false, its returned like hex. 
    
corePlugins: {
     backgroundOpacity: false,
    }

**Reply by Noitidart** [reply]
Thanks for sharing this! I was wondering how to do this too because the rgb convention they use is rgb with slash and opacity which is not supported in some browsers - https://stackoverflow.com/q/71227760/1828637

Where did you find the backgroundOpacity name? I also want to do same for text-color

**Reply by Noitidart** [reply]
Oh found it, it was `textOpacity` - https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js

There are many other `*Opacity` things, ideally instead of using rgb/alpha they use rgba convention. I wonder why they did that. I'll post it as bug, as using rgba supports way more browsers. Was browser support also your issue?

**Reply by Noitidart** [reply]
Posted issue - https://github.com/tailwindlabs/tailwindcss/issues/7618

**Reply by musen34** [reply]
> #7618

I want that my website is should be loading fast and scrolling good also low-end devices. And now, its loading/scrolling slowly according to pure css.. I've tested Galaxy S3 mini, S3 mini plus and some similar low-end devices. And thats why Im searching cause of slow.
